### PR TITLE
Documentation Update

### LINKED ISSUE #118: Documentation
The README is missing the following documentation:
- [ ] How to set up a local dev environment
- [ ] Conventions
- [ ] Project structure, file structure, libraries used

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
Closes #118

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,116 @@
...
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGMENTS -->
+## Acknowledgments
+* [Choose an Open Source License](https://choosealicense.com)
+* [Img Shields](https://shields.io)
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+> Please update this documentation if you make changes to the KWoC frontend or any other part of KWoC which affects the frontend. Future humans will praise you.
```
Comment (proffapt) [inline README.md:116]: Keep it generic don't mention about frontend specifically, just to stay in sync with the template we will be making. Uniformity is the first step in decluttering.

**Review by proffapt — CHANGES_REQUESTED** [review]
I would recommend to keep things like
- Contributing.md
- COC.md
etc.. except LICENSE.txt (coz it won't work) into .github folder.. it just reduces the clutter in front + still follows good practices

image
image
License is not being recognised for some reason?

File: README.md (hunk 1 above)
Comment (harshkhandeparkar) [reply README.md:116]: 👀 
Suggest a new sentence please, sir.

**Comment by harshkhandeparkar** [comment]
> I would recommend to keep things like
> 
>     * Contributing.md
> 
>     * COC.md
>       etc.. except LICENSE.txt (coz it won't work) into .github folder.. it just reduces the clutter in front + still follows good practices
> 
> 
> image image License is not being recognised for some reason?

There is no license yet :harold:

**Comment by proffapt** [comment]
why tf do i see [this](https://github.com/kossiitkgp/KWoC-Frontend/blob/documentation/LICENSE)?!?

**Comment by harshkhandeparkar** [comment]
> why tf do i see [this](https://github.com/kossiitkgp/KWoC-Frontend/blob/documentation/LICENSE)?!?

because it is in the `documentation` branch?

**Comment by harshkhandeparkar** [comment]
> why tf do i see [this](https://github.com/kossiitkgp/KWoC-Frontend/blob/documentation/LICENSE)?!?

because

> why tf do i see [this](https://github.com/kossiitkgp/KWoC-Frontend/blob/documentation/LICENSE)?!?

because it is in the `documentation` branch?

**Comment by harshkhandeparkar** [comment]
> into .github folder.. it just reduces the clutter in front + still follows good practices

We only have a CONTRIBUTING.md so ig we'll let it be in the root for now. The main clutter is due to other files which we can't move yet.

File: README.md (hunk 1 above)
Comment (harshkhandeparkar) [reply README.md:116]: If you don't have a suggestion, I believe we can resolve this :slightly_smiling_face:

Comment (harshkhandeparkar) [reply README.md:116]: Also we can make this sentence the template, just replacing KWoC Frontend with whichever project

**Comment by proffapt** [comment]
> because it is in the documentation branch?

Sirf main branch ka dikhta hai kya? ya fir license ki link mein main branch hogi ig.. nvm

> We only have a CONTRIBUTING.md so ig we'll let it be in the root for now. The main clutter is due to other files which we can't move yet.

This repository doesn't have now, but it will have soon, because template will have..

File: README.md (hunk 1 above)
Comment (harshkhandeparkar) [reply README.md:116]: repository -> project

**Comment by harshkhandeparkar** [comment]
> Sirf main branch ka dikhta hai kya? ya fir license ki link mein main branch hogi ig.. nvm

sirf default ka dikhta hai ofc

> This repository doesn't have now, but it will have soon, because template will have..

tab dekhte hain. sabka ek saath change karenge

File: README.md [hunk 2]
```diff
@@ -1,37 +1,117 @@
...
+
+<!-- ACKNOWLEDGMENTS -->
+## Acknowledgments
+* [Choose an Open Source License](https://choosealicense.com)
+* [Img Shields](https://shields.io)
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+***
+> Please update this documentation if you make changes to this project or any other part that may affect this. Future humans will praise you.
```
Comment (proffapt) [inline README.md:117]:
```suggestion
> Please remember to update this documentation whenever changes are made to this project or any other relevant project that may affect this one. Future humans will praise you.
```

This looks better imo.

File: README.md (hunk 1 above)
Comment (proffapt) [reply README.md]: shifted to another review section. resolving this.

File: README.md (hunk 2 above)
Comment (harshkhandeparkar) [reply README.md]: Lo bhaiyya kar diya

**Comment by proffapt** [comment]
Ab baaki content bhi daaldo, baadka kyun wait karna.. COC, guidlines security whatever..

If you would prefer take them from my reamde..

**Comment by harshkhandeparkar** [comment]
> 

bhai ye sab karne ka samay nahi hai mere paas. COC vagera baad mein decide kar lenge. aur bhi repos hain. abhi frontend pe kaam shuru karna hai.

**Review by proffapt — APPROVED** [review]
LGTM.

**Comment by proffapt** [comment]
merge kardu?

**Comment by harshkhandeparkar** [comment]
> merge kardu?

kardo. thanks.

**Pull request merged**